src/wallet/api.rs

```diff
@@ -1255,63 +1260,97 @@ impl Wallet {
         let target_weight = Weight::from_vb_unwrap(
             (Amount::SIZE + VarInt::from(P2WPKH_SPK_SIZE).size() + P2WPKH_SPK_SIZE) as u64,
         );
-        let avg_output_weight = (change_weight.to_wu() + (target_weight.to_wu())) / 2;
+        let avg_output_weight = (change_weight.to_wu() + target_weight.to_wu()) / 2;
         let avg_input_weight = unspents
             .iter()
             .map(|(_, spend_info)| {
                 // Base weight of the input (OutPoint + sequence + empty script_sig)
-                let base_weight = 32 + 4 + 4 + 1; // 32 bytes for OutPoint, 4 bytes for sequence,4 bytes for vout, 1 byte for empty script_sig
+                let input_base_weight = 32 + 4 + 4 + 1; // OutPoint + sequence + vout + empty script_sig
```

Comment:
The `target_weight`, `change_weight` and `input_base_weight` can be turned into constants, with comments mentioning where they are coming from. Having them in code is too verbose. It was like this from start, we can clean it up now.

Reply:
Theres a TODO marked on it, we can fix it here as we are touching the code anyway.